bin/install-dependencies.sh

```diff
@@ -94,6 +98,9 @@ case "${OSTYPE}" in
         ERLANGVERSION=${ERLANGVERSION} ${SCRIPTPATH}/yum-erlang.sh
         ELIXIRVERSION=${ELIXIRVERSION} ${SCRIPTPATH}/install-elixir.sh
       fi
+      if [[ ! ${SKIPCLOUSEAU} ]]; then
+	CLOUSEAUVERSION=${CLOUSEAUVERSION} ${SCRIPTPATH}/install-clouseau.sh
```

Comment:
Do we want to set a default version too, like in L32-L34? `CLOUSEAUVERSION=${CLOUSEAUVERSION:-x.y.z}`

Reply:
Right, nice catch!